apiserver/facades/client/modelmanager/modelmanager_test.go

```diff
@@ -515,38 +590,27 @@ func (s *modelManagerSuite) TestCreateModelNoDefaultCredentialNonAdmin(c *gc.C)
 	c.Assert(err, gc.ErrorMatches, "no credential specified")
 }
 
-func (s *modelManagerSuite) TestCreateModelUnknownCredential(c *gc.C) {
```

Comment:
Should we keep this test?

Reply:
This is just testing an error from state, thats why I removed it.

Reply:
I'll add it back in and skip it instead, in case whoever touches it wants to just update it